Jenkinsfile

```diff
@@ -1,43 +0,0 @@
-node {
```

Comment:
Is this part of the PR?

Reply:
I didn't want to possibly retrigger builds that mess up `staging`/`testing` since we still haven't verified that the Jenkins recycle fixed the issue with the Jenkins job.

Reply:
Also I found a bug so glad it didn't go in yet!